README.md

```diff
@@ -36,8 +36,6 @@ The release can be found here: https://github.com/PDFTron/pdftron-react-native/r
 
 ## Installation
 
-0. If using yarn, do: `yarn global add react-native-cli`
```

Comment:
Is this no longer needed?

Reply:
yeah, it works either way (added or not added)